diesel/src/associations/mod.rs

```diff
@@ -268,6 +271,8 @@
 //! Explicit type annotations have been added
 //! to make each line a bit more clear.
 //!
+//! [`grouped_by`]: trait.groupedby.html#tymethod.grouped_by
```

Comment:
Do we really need to add that link three times?

Reply:
Hmm, when I read the initial code, I remember seeing links that were already duplicated, and thus assumed the style was to put the links after each paragraph where they're used, to keep them close to the actual link. However when I look at the source again, it seems this isn't actually what happened.